### DISCUSSION TITLE
Method to detect when user ends onPanDrag

### DISCUSSION THREAD

**Discussion by jascenci** [react-native-maps/react-native-maps#4385] [Q&A]
I'm looking for a solution to detect when a user has ended onPanDrag. Essentially lifting their finger. I'm drawing a polyline on the map and want to detect when a user has finished drawing to convert the Polyline coordinates into a Polygon. Unfortunately react-native-maps does not have an event like onPanDragEnd.

**Comment by Kameal22** [comment]
Yeah, how to achieve that?

**Comment by jan-kozinski** [comment]
Hey, the library doesn't supoort such API yet, but we can add it soon
